win32port/zlib/gzguts.h

```diff
@@ -12,7 +12,7 @@
 #  endif
 #endif
 
-#if ((__GNUC__-0) * 10 + __GNUC_MINOR__-0 >= 33) && !defined(NO_VIZ)
+#if ((__GNUC__-0) * 10 + __GNUC_MINOR__-0 >= 33) && !defined(NO_VIZ) && !defined(_WIN32)

```

Comment:
This seems dodgy, having to change in zlib why is it needed? Are we using something internal to ZLIB somewhere?